proto/buf/alpha/registry/v1alpha1/repository_tag.proto

```diff
@@ -65,6 +84,19 @@ message CreateRepositoryTagResponse {
   RepositoryTag repository_tag = 1;
 }
 
+message UpdateRepositoryTagRequest {
+  // The ID of the repository this tag belongs to.
+  string repository_id = 1;
+  // The name of the repository tag, to update e.g. "6e2e7f24718a76caa32a80d0e2b1841ef2c61403".
+  string name = 2;
+  // The name of the commit this tag should reference.
+  optional string commit_name = 3;
```

Comment:
In which scenario is this optional?

Reply:
This is so that we're consistent with other `Update*` RPCs, where thing should be optional.
There could be other things to update here, concievably.

Reply:
Currently the only thing that can be updated in a tag is the commit its pointing to. Having it optional is super confusing. If we're leaving it optional, I guess in the backend if this is not passed this is a no-op?

Reply:
Yes. This is also consistent with v1beta1: https://github.com/bufbuild/registry-proto/blob/d73fe14dfdc908ab9dffde2447f13b1c570098a1/buf/registry/module/v1beta1/tag_service.proto#L135